e2e-tests/tfm/cypress/e2e/journeys/case-amendments/facility-end-date/bss-deal-expect-no-facility-end-date-features.spec.js

```diff
@@ -0,0 +1,77 @@
+import relative from '../../../relativeURL';
+import facilityPage from '../../../pages/facilityPage';
+import amendmentsPage from '../../../pages/amendments/amendmentsPage';
+import MOCK_DEAL_AIN from '../../../../fixtures/deal-AIN';
+import dateConstants from '../../../../../../e2e-fixtures/dateConstants';
+import { ADMIN, BANK1_MAKER1, PIM_USER_1 } from '../../../../../../e2e-fixtures';
+
+context('Amendments - BSS/EWCS deal does not display any Facility end date pages or fields', () => {
+  let dealId;
+  const facilities = [];
+
+  before(() => {
+    cy.insertOneDeal(MOCK_DEAL_AIN, BANK1_MAKER1).then((insertedDeal) => {
+      dealId = insertedDeal._id;
+
+      const { dealType, mockFacilities } = MOCK_DEAL_AIN;
+
+      cy.createFacilities(dealId, [mockFacilities[0]], BANK1_MAKER1).then((createdFacilities) => {
+        facilities.push(...createdFacilities);
+      });
+
+      cy.submitDeal(dealId, dealType, PIM_USER_1);
+    });
+  });
+
+  beforeEach(() => {
+    cy.login(PIM_USER_1);
+    const facilityId = facilities[0]._id;
+    cy.visit(relative(`/case/${dealId}/facility/${facilityId}`));
+  });
+
+  after(() => {
+    cy.deleteDeals(dealId, ADMIN);
+    facilities.forEach((facility) => {
+      cy.deleteFacility(facility._id, BANK1_MAKER1);
+    });
+  });
+
+  it('should not show any facility end date fields in the Details tab', () => {
+    facilityPage.facilityIsUsingFacilityEndDate().should('not.exist');
+    facilityPage.facilityFacilityEndDate().should('not.exist');
+    facilityPage.facilityBankReviewDate().should('not.exist');
+  });
+
+  it('should go straight from the cover end date amendments page to the review amendment answers page without displaying any facility end date fields', () => {
+    facilityPage.facilityTabAmendments().click();
+    amendmentsPage.addAmendmentButton().should('exist');
+    amendmentsPage.addAmendmentButton().contains('Add an amendment request');
```

Comment:
is there a content string for this?

Reply:
No there isn't -- I expect it relatively static and only actually repeated in tests so is not worth it imo (due to the size of pr im being really stingy on scope creep, sorry!)